service/surf/apps/materials/jinja2/portal/index.html

```diff
@@ -2,15 +2,15 @@
 <html>
 <head>
 
-    <title>SURF | edusources</title>
+    <title>{{ meta_title }}</title>
```

Comment:
Verandert dit nu mee als je navigeert?

Reply:
Nee, daar is een nieuwe ticket voor (doet het ook al niet op prod en dit is het minimale voor Google)